pkg/common/config/aws_config.go

```diff
@@ -26,6 +46,59 @@ var (
 )
 
 func InitAWSViper() {
+	_ = viper.BindEnv(AWSSharedCredentials, "AWS_SHARED_CREDENTIALS")
+	RegisterSecret(AWSSharedCredentials, "aws-shared-credentials")
+
+	viper.SetDefault(AWSCredentialsFile, "/tmp/.aws/osde2e/credentials")
```

Comment:
Is this default created automatically somewhere? `/tmp` is cleared on every restart so it may be better to put this in the `$HOME/.aws` directory